components/automate-ui/src/app/entities/client-runs/client-runs.requests.spec.ts

```diff
@@ -80,10 +80,11 @@ describe('ClientRunsRequests', () => {
       const filters: NodeFilter = <NodeFilter>{
         page: 0,
         pageSize: 100,
-        organizations: ['org1'],
-        servers: ['chef.org'],
         status: 'success',
-        searchBar: [{'text': 'bob', 'type': 'name'}]
+        searchBar: [
+          {'text': 'bob', 'type': 'name'},
+          {'text': 'chef.org', 'type': 'source_fqdn'},
+          {'text': 'org1', 'type': 'organization'}]
```

Comment:
The `organizations` and `servers` fields were removed to only use the `searchBar` field.